crates/bevy_render/src/camera/camera.rs

```diff
@@ -136,7 +136,7 @@ impl Camera {
     #[inline]
     pub fn to_logical(&self, physical_size: UVec2) -> Option<Vec2> {
         let scale = self.computed.target_info.as_ref()?.scale_factor;
-        Some((physical_size.as_dvec2() / scale).as_vec2())
+        Some((physical_size.as_dvec2() / scale as f64).as_vec2())
```

Comment:
```suggestion
        Some(physical_size.as_vec2() / scale)
```
Does something like this work instead? I think we can simplify this by just swapping the physical size to f32 at the start.